octodns/provider/ns1.py

```diff
@@ -216,6 +233,12 @@ class Ns1Provider(BaseProvider):
         # Only required if using dynamic records
         monitor_regions:
           - lga
+        # Optional. Default: false. true is Recommended, but not the default
+        # for backwards compatibility reasons. If true, all NS1 monitors will
+        # use a shared notify list rather than one per record & value
+        # combination. See CHANGELOG for more information before enabling this
```

Comment:
Not super critical, but it'd be nice and convenient to have a direct link to the CHANGELOG.

Reply:
I can link the doc, but not to the exact section or anything since it's not yet merged into a permanent location.

Reply:
Yeah makes sense. I guess anyone reading these code comments knows their way to the CHANGELOG file.